src/Repec.php

```diff
@@ -554,6 +567,8 @@ public function getEntityTemplate(ContentEntityInterface $entity) {
       }
     }
 
+    $this->moduleHandler->alter('repec_template', $template, $entity);
```

Comment:
This isn't what I meant. 

`hook_repec_{SERIE}_template_alter' could stay for altering a single template, but also, in addition, have a `hook_repec_templates_alter' that takes an array of templates. If that doesn't make sense, then `hook_repec_template_alter` would be fine as well.

Reply:
I see.. Okay, that sounds good and cleaner. I will update the code. I will add both the hooks. I will refer `hook_form_alter()`/`hook_form_FORM_ID_alter()` for the invocation order.

Reply:
Done. Thanks!